src/utils/request.ts

```diff
@@ -14,8 +15,12 @@ const options = {
 export const httpAgent = new http.Agent(options);
 export const httpsAgent = new https.Agent(options);
 
-export async function request(url, opts) {
-  return await got(url, {
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
not sure if we need this lint rule anywhere

Reply:
It's a matter of strictness, in the crawler we got hit by strong bug because typescript is good a guessing the return type and does not warn you if an other function expected a "compatible but slightly different type".